site/realm/functions/logIncidentHistory.js

```diff
@@ -0,0 +1,31 @@
+
+/**
+ * This function creates a entry into the Incident History collection
+ * 
+ * @param {object} input A JSON object that contains the Incident data
+ * 
+*/
+exports = async (input) => {
+
+  const incidentsHistory = context.services.get('mongodb-atlas').db('history').collection("incidents");
+
+  const {
+    AllegedDeployerOfAISystem,
+    AllegedDeveloperOfAISystem,
+    AllegedHarmedOrNearlyHarmedParties,
+    ...incident } = input;
+
+  incident['Alleged deployer of AI system'] = AllegedDeployerOfAISystem;
+  incident['Alleged developer of AI system'] = AllegedDeveloperOfAISystem;
+  incident['Alleged harmed or nearly harmed parties'] = AllegedHarmedOrNearlyHarmedParties;
+
+  await incidentsHistory.insertOne(incident);
```

Comment:
The payload only expects a `incident_id` but this assumes the entire incident is being sent 🤔

Reply:
The payload is the returned value, the Input type is: `"input_type": "History_incidentInsertInput",`